### PR TITLE
Clean dep

### PR DESCRIPTION
**Pull request by Joseph-ljx**
1. Clean Redundant Dependencies
2. Create New Front End Layout
3. Create Best Sellers
4. fix readMe commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 - Build react app
   - run `npm run build`
 - Build docker image:
-  - run `docker build -t krispykopters/<unique tag> .`
+  - run `docker build . -t krispykopters/<unique tag> .`
```
Comment (JunTaoLuo) [inline README.md:86]: There is a dot at the end of the command already, adding another one would break the command right?

**Review by JunTaoLuo — COMMENTED** [review]
Looks good but you'll have to rebase on the latest changes

File: README.md (hunk 1 above)
Comment (Joseph-ljx) [reply README.md:86]: Oh, but the first command does not work in my environment. It works after i add the dot. Does the first one work in yours? If yes, ommit this change.

**Comment by Joseph-ljx** [comment]
yep, I will try to rebase it tonight. And also cuple of more changes.

**Pull request closed without merging**